bepdf/beos/BepdfApplication.cpp

```diff
@@ -545,12 +545,21 @@ void BepdfApplication::RefsReceived ( BMessage * msg )
 	BString ownerPassword, userPassword;
 	const char *owner = NULL;
 	const char *user  = NULL;
+    int32 pageNum = 0;
+
 	if (B_OK == msg->FindString("ownerPassword", &ownerPassword)) {
 		owner = ownerPassword.String();
 	}
 	if (B_OK == msg->FindString("userPassword", &userPassword)) {
 		user = userPassword.String();
 	}
+    status_t result = msg->FindInt32(PAGE_NUM_MSG, &pageNum);
+    if (result != B_OK) {
+        if (result != B_NAME_NOT_FOUND) {
+            BAlert *error = new BAlert(B_TRANSLATE("Error"), B_TRANSLATE("BePDF: Error getting page number!"), B_TRANSLATE("Close"), NULL, NULL, B_WIDTH_AS_USUAL, B_WARNING_ALERT);
```

Comment:
Alerts normally don't start with the app's name, here "BePDF:".
The starndard button label to acknowledge an alert is "OK".